src/content/pages/volunteers.mdx

```diff
@@ -38,83 +44,86 @@ This is a very people-first position and will require spending a considerable ti
+* Ensuring people are seated before a talk, and the room is not overcrowded
+* Making sure the session chair and speakers are in the room on time
 * Keeping in touch with the AV team to ensure everything is in order
-* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed
+* Keeping the [Code of Conduct (CoC)](https://www.europython-society.org/coc/) in mind and contact CoC committee members if needed
 
 We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.
 
 ## Session Chairing
 
 A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:
 
-* Helping speakers in preparing their computers for the talks
+* Helping speakers prepare their computers for the talks
 * Introducing the speakers to the audience
-* Signalling time left to the speakers
+* Signalling the speakers how much time is left
 * Making sure talks start and finish on time
-* Moderating Q&A sessions, assisting with passing the microphones and read out questions from remote attendees
+* Moderating Q&A sessions, assisting with passing the microphones and reading out questions from remote attendees
 
 ## Setup/Teardown
 
-* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)
-* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward
-* This job involves walking and carrying items, some of which may be heavy.
+* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare, etc.)
+* On the last day (Friday afternoon for the main conference, Sunday for the sprints), there will be a venue cleanup afterwards
+* This job involves walking and carrying items, some of which may be heavy
 
-## Greeter / Badge check
+## Greeter / Badge Check
 
-* Checking attendees on the venue for badges.
-* Send people to the registration desk if they don't have one and are not from the venue (e.g. AV team, catering, etc.).
-* Report people that don't have badges and don't want to go to the registration desk.
+* Checking attendees at the venue for badges
+* Sending people to the registration desk if they don't have a badge, and not from the venue (e.g. AV team, catering, etc.)
+* Reporting people who don't have badges and refuse to go to the registration desk
 
 ## Runners and Ad-hoc Activities
 
 There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):
 
-* Checking rooms to ensure all the necessary resources are in place
-* Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties
+* Checking the rooms to ensure all the necessary resources are in place
```

Comment:
If we add "the", is should be "Check"

Reply:
The presence of "the" doesn't change the verb from "checking" to "check". Both "checking the rooms" and "checking rooms" are correct, but you're right. We shouldn't use "the" because we haven't mentioned any specific rooms to be checked.